### PR TITLE
Gherkin: QuickStarts-enhanced-capabilities

### PR DESCRIPTION
**Pull request by sanketpathak**
Epic: https://issues.redhat.com/browse/ODC-5010
Story: https://issues.redhat.com/browse/ODC-5432

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_Access_a _code_snippet.feature [hunk 1]
```diff
@@ -0,0 +1,71 @@
+Feature: Add ability to filter Quick Starts catalog
+              As a user, i want to be able to filter Quick Starts in the catalog by status of the Quick Starts and search Quick Starts catalog by name, description, tags and prerequisites.
```
Comment (rohitkrai03) [inline frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_Access_a _code_snippet.feature:2]: Isn't this about copy to clipboard feature?

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_Access_a _code_snippet.feature (hunk 1 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_Access_a _code_snippet.feature:2]: Sorry about that, updated

File: frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_Access_a _code_snippet.feature [hunk 2]
```diff
@@ -0,0 +1,72 @@
+Feature: Access a code snippet from a quick start
+              As a user, I want to be able to visualize seeing a CLI snippet in Quick Starts differently, able to copy it and f WTO is installed, and if Web Terminal is installed, I should have an option to execute the command directly without having to copy/paste
```
Comment (rohitkrai03) [inline frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_Access_a _code_snippet.feature:2]:
```suggestion
              As a user, I want to be able to visualize seeing a CLI code snippet in Quick Starts differently, able to copy it and if Web Terminal is installed, I should have an option to execute the command directly without having to copy/paste.
```

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

**Review by makambalaji — CHANGES_REQUESTED** [review]
Execute 'Yarn run gherkin-lint' for this features and fix the linter issues

File: frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_Access_a _code_snippet.feature (hunk 2 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_Access_a _code_snippet.feature:2]: Updated

**Review by makambalaji — COMMENTED** [review]
Please add epic number and @to-do for automation scenarios

**Comment by makambalaji** [comment]
@sanketpathak - Need approval or lgtm from DEV team. Before QE final approval
cc: @rohitkrai03

**Review by rohitkrai03 — APPROVED** [review]

**Pull request merged**